hack/lint-manifests.sh

```diff
@@ -21,6 +21,6 @@ DIFF=$(diff -q $TMP_DIR ${OPERATOR_DIR}/config/crd/bases)
 if [ -n "${DIFF}" ]; then
     echo -e "${RED}x manifests linting failed${NC}"
     echo -e "Generated manifests in config/crd/bases are not on par with API types in apis/telemetry/v1alpha1"
-    echo -e "Please run 'make manifests-local'"
+    echo -e "Please run 'make manifests'"
```

Comment:
We don't have manifests-local anymore, do we?

Reply:
yes, that's why we removed it :D